doc/release-notes/upgrading.en.rst

```diff
@@ -133,10 +135,25 @@ The following :file:`records.yaml` changes have been made:
   files, but they did not have any effect.  These have been removed from default configs files.
 - The default values for ``proxy.config.http.request_header_max_size``, ``proxy.config.http.response_header_max_size``, and
   ``proxy.config.http.header_field_max_size`` have been changed to 32KB.
+- The records.yaml entry ``proxy.config.http.server_ports`` now also accepts the
+- The default values for :ts:cv:``proxy.config.http.request_header_max_size``, :ts:cv:``proxy.config.http.response_header_max_size``, and
+  :ts:cv:``proxy.config.http.header_field_max_size`` have been changed to 32KB.
+- The records.yaml entry :ts:cv:``proxy.config.http.server_ports`` now also accepts the
+  ``allow-plain`` option
+- The records.yaml entry ``proxy.config.http.down_server.abort_threshold`` has
+  been removed
+- The records.yaml entry :ts:cv:``proxy.config.http.cache.max_open_write_retry_timeout`` has been added to specify a timeout for starting a write to cache
```

Comment:
just got caught on my pr with this too, it seems the format is :
```
:ts:cv:`var_name`
```
single  single-quote quote after the `:`